StripeCore/StripeCore/Source/Analytics/STPAnalyticEvent.swift

```diff
@@ -149,6 +149,7 @@ import Foundation
     case link2FAComplete = "link.2fa.complete"
     case link2FACancel = "link.2fa.cancel"
     case link2FAFailure = "link.2fa.failure"
+    case linkBailedToWebFlow = "link.native.bailed"
```

Comment:
nit: case should be named `linkNativeBailed`

Reply:
Fixed!